airflow/bin/cli.py

```diff
@@ -363,85 +363,7 @@ def run(args, dag=None):
-            # out.  If the previous version is not zombied out, it is still possible that the scheduler bounced, and
-            # incorrectly thought this task was an orphan, in which case the previous version will eventually reclaim
-            # it.
-            from airflow.jobs import LocalTaskJob as LJ
-            TI = TaskInstance
-            secs = conf.getint('scheduler', 'scheduler_zombie_task_threshold')
-            limit_dttm = datetime.utcnow() - timedelta(seconds=secs)
-            log.info("Task id is %s job id is %s", ti.task_id, ti.job_id)
-            latest_heartbeat = session.query(LJ).filter(LJ.id == ti.job_id).first().latest_heartbeat
-            if latest_heartbeat >= limit_dttm:
-
-                real_ti = session.query(TI).filter(
-                    TI.dag_id == ti.dag_id,
-                    TI.task_id == ti.task_id,
-                    TI.execution_date == ti.execution_date
-                ).with_for_update().first()
-
-                if real_ti.state == State.RUNNING:
-                    log.info(
-                        "For submitted task \"{}\" the hostname was set, and the heartbeat indicates that the previous "
-                        "worker is still running.  Additionally, the state of the task has been set to \"running\" "
-                        "which indicates that the previous worker has re-asserted ownership over this task.  "
-                        "Exiting.".format(
-                            ti.task_id
-                        )
-                    )
-                    session.merge(real_ti)
-                    session.commit()
-                    return
-                else:
-                    log.info(
-                        "For submitted task \"{}\" the hostname was set, and the heartbeat indicates that the previous "
-                        "worker could still be running.  The state of the task instance has not been set to "
-                        "\"running\" yet so we're setting the task state to None to give the previous worker time to "
-                        "re-assert ownership over this task.  Exiting.".format(
-                            ti.task_id
-                        )
-                    )
-                    real_ti.state = State.NONE
-                    session.merge(real_ti)
-                    session.commit()
-                    return
-            else:
-                log.info(
-                    "For submitted task \"{}\" the hostname was set, but the heartbeat indicates that the previous "
-                    "worker has died.  Proceeding with task run.".format(
-                        ti.task_id
-                    )
-                )
-        else:
-            log.info(
-                "For submitted task \"{}\" the hostname was not set, which implies that there exists no active "
-                "previous worker.  Proceeding with task run.".format(
-                    ti.task_id
-                )
-            )
```

Comment:
This is all just reverting changes I made to deal with issues introduced by the old orphan adoption mechanism.